tests/browser/test/basic_playground_test.js

```diff
@@ -129,6 +136,27 @@ suite('Right Clicking on Blocks', function (done) {
     chai.assert.isTrue(isEnabled);
   });
 
+  test('Add Comment', async function () {
+    const block = await dragNthBlockFromFlyout(browser, 'Loops', 0, 20, 20);
+
+    const blockId = block.id;
+    await contextMenuSelect(browser, block, 'Add Comment');
+
+    const commentText = await getCommentText(browser, blockId);
+    chai.assert.equal(commentText, '');
+  });
+
+  test('Remove Comment', async function () {
+    const block = await dragNthBlockFromFlyout(browser, 'Loops', 0, 20, 20);
+
+    const blockId = block.id;
+    await contextMenuSelect(browser, block, 'Add Comment');
```

Comment:
Christopher and I discussed in a different PR that since this is actually part of setup, it would be better to move it into the previous chunk of code (under `const block = await`) and use Blockly's APIs rather than the context menu.

Reply:
Do you mean that I should use json to load a block in the correct location, with the correct value for enabled/comment/collapsed, rather than dragging in?

Reply:
In [my other PR Christopher suggested](https://github.com/google/blockly/pull/7279#discussion_r1261040532) using `browser.execute` to disable it. But you're right it does seem a bit weird to drag out the block using webdriver, but disable it using the API. So up to you! I'll fix my PR to match if necessary.